test/ctia/entity/incident_test.clj

```diff
@@ -572,3 +572,57 @@
                           (is (= (normalize [incident1 incident3])
                                  (normalize parsed-body)))))))
                (finally (purge-incidents! app))))))))
+
+(deftest filter-incidents-by-scores-range
+  (es-helpers/for-each-es-version
+    "filter by scores"
+    [7]
+    #(ductile.index/delete! % "ctia_*")
+    (helpers/with-properties (-> ["ctia.auth.type" "allow-all"]
+                                 (into es-helpers/basic-auth-properties)
+                                 (conj "ctia.http.incident.score-types" "asset,ttp"))
+      (helpers/fixture-ctia-with-app
+        (fn [app]
+          ;(helpers/set-capabilities! app "foouser" ["foogroup"] "user" all-capabilities)
+          ;(whoami-helpers/set-whoami-response app "45c1f5e3f05d0" "foouser" "foogroup" "user")
+          (try (let [incident1 (assoc (gen-new-incident)
+                                      :title "incident1"
+                                      :assignees ["assignee1"]
+                                      :scores {:ttp 30
+                                               :asset 100})
+                     incident2 (assoc (gen-new-incident)
+                                      :title "incident2"
+                                      :scores {:ttp 50
+                                               :asset 50})
+                     incident3 (assoc (gen-new-incident)
+                                      :title "incident3"
+                                      :scores {:ttp 70
+                                               :asset 0})
+                     normalize (fn [incidents]
+                                 (->> incidents
+                                      (map #(select-keys % [:title :scores]))
+                                      (sort-by :title)))]
+                 (create-incidents app #{incident1 incident2 incident3})
+                 (testing "50<=ttp"
+                   (let [{:keys [parsed-body] :as raw} (search-th/search-raw app :incident {:query "scores.ttp:>=50"})]
+                     (and (is (= 200 (:status raw)) (pr-str raw))
+                          (is (= (normalize [incident2 incident3])
+                                 (normalize parsed-body))
+                              (pr-str parsed-body)))))
+                 (testing "ttp<=50"
+                   (let [{:keys [parsed-body] :as raw} (search-th/search-raw app :incident {:query "scores.ttp:<=50"})]
+                     (and (is (= 200 (:status raw)) (pr-str raw))
+                          (is (= (normalize [incident1 incident2])
+                                 (normalize parsed-body))))))
+                 (testing "50<=ttp && asset<=60"
+                   (let [{:keys [parsed-body] :as raw} (search-th/search-raw app :incident {:query "(scores.ttp:<=50) AND (scores.asset:>=60)"})]
```

Comment:
You could you this equivalent range syntax
```suggestion
                   (let [{:keys [parsed-body] :as raw} (search-th/search-raw app :incident {:query "(scores.ttp:[50 TO 60]"})]
```

Reply:
Ah, this is actually 2 different scores, ttp and asset.